02Lesson/02Project/index.html

```diff
@@ -1,10 +1,66 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Daily Planet</title>
   <link rel="stylesheet" href="css/style.css"/>
 </head>
-<body>
-  <!-- You code start here -->
+<body background="https://static1.squarespace.com/static/550890c6e4b076f89f2184cf/t/5589d2b5e4b0872e7abdef3e/1435095762334/Austin+Cityscape+and+Lady+Bird+Lake+2.jpg?format=1500w">

```

Comment:
Attribute background is obsolete in HTML5. Use CSS property `background-image` instead. W3C validator in atom should point this out too.